cmd/cloud-controller-manager/.import-restrictions

```diff
@@ -1,84 +0,0 @@
-rules:
```

Comment:
rather than eliminating this file, can we change it to this to ensure no new k8s.io/kubernetes deps get added?

```
  - selectorRegexp: k8s[.]io/kubernetes
    allowedPrefixes:
      - k8s.io/kubernetes/cmd/cloud-controller-manager
```

Reply:
+1